go/systests/multiuser_common_test.go

```diff
@@ -296,6 +296,14 @@ func (u *smuUser) signup() {
 	backupKey = backups[0]
 	backupKey.secret = signupUI.info.displayedPaperKey
 	u.backupKeys = append(u.backupKeys, backupKey)
+
+	// Reconfigure config subsystem in Primary Global Context and also
+	// in all clones. This has to be done after signup because the
+	// username changes, and so does config filename.
+	dw.tctx.G.ConfigureConfig()
+	for _, clone := range dw.clones {
+		clone.G.ConfigureConfig()
+	}
```

Comment:
I am now onboard with this.  I think the other tests that used the `installKeybase` then `signup` path just lucked out and never hit a server-side codepath where the Session was being loaded.  But this test does.